### DISCUSSION TITLE
Avoid Crossing Perimeters Bug?

### DISCUSSION THREAD

**Discussion by buildjournal** [supermerill/SuperSlicer#2466] [General]
In CURA there's a setting called Avoid Printed Parts so the head moves around the printed parts. Looks like SS's version is called "Avoid Crossing Perimeters, but it just does not work. The print preview shows travel across perimeters and nothing changes. Is anyone else having issues? Link to screenshots below.

https://imgur.com/a/dvXBE5s

**Comment by supermerill** [comment]
How can you print 2 objects without at least a move in each direction?
